### DISCUSSION TITLE
Deploying Knowledge platform in AWS

### DISCUSSION THREAD

**Discussion by aastarurmika** [project-sunbird/sunbird-devops#2207] [Q&A]
Hi, 
We are trying to deploy KP to AWS and have hit a roadblock on how to use s3 instead of Azure blob storage. Can you please help us with this.

SB Version: 3.5

**Comment by rjshrjndrn** [comment]
Hi @aastarurmika, Welcome to the Sunbird community. We've all the installation-related queries answered in our community forum. Please raise an issue there. 
https://github.com/project-sunbird/sunbird-community/discussions/categories/-20-sunbird-installation-and-configuration
